src/app/components/itinerary/itinerarylist/itinerarylist.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { Itinerary } from 'src/app/models/itinerary';
+import { ItineraryService } from 'src/app/services/itinerary.service';
+
+@Component({
+  selector: 'app-itinerarylist',
+  templateUrl: './itinerarylist.component.html',
+  styleUrls: ['./itinerarylist.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
  styleUrls: ["./itinerarylist.component.css"]
```